.github/workflows/build_and_test.yml

```diff
@@ -623,6 +623,12 @@ jobs:
     - name: Run documentation build
       run: |
         cd docs
+        if [ -f "./dev/is-changed.py" ]; then
+          # Skip PySpark and SparkR docs while keeping Scala/Java/SQL docs
+          pyspark_modules=`cd dev && python -c "import sparktestsupport.modules as m; print(','.join(m.name for m in m.all_modules if m.name.startswith('pyspark')))"`
+          if [ `./dev/is-changed.py -m $pyspark_modules` = false ]; then export SKIP_PYTHONDOC=1; fi
+          if [ `./dev/is-changed.py -m sparkr` = false ]; then export SKIP_RDOC=1; fi
+        fi
```

Comment:
This logic itself is identical with the current one used in job skipping.

https://github.com/apache/spark/blob/2440b6b731758308d0db88c10edb5b9a71d81c92/.github/workflows/build_and_test.yml#L84-L90